uelc/main/views.py

```diff
@@ -208,7 +208,10 @@ def notify_facilitators(self, request, path, notification):
                  content=json.dumps(msg))
 
         socket.send(json.dumps(e))
-        socket.recv()
+        try:
+            socket.recv(zmq.NOBLOCK)
+        except zmq.ZMQError:
+            pass
```

Comment:
Nick, thanks for the heads up. I don't think this will cause any problems at all. I do wonder however if we need to handle to error better...